.code-samples.meilisearch.yaml

```diff
@@ -1065,3 +1065,21 @@ filtering_update_settings_1: |-
       "genres",
       "director"
     ]'
+fetch_documents_1: |-
```

Comment:
Following the same logic as the naming for get and post search, I think it is more understandable if you name this sample `get_documents_post_1`. `fetch_documents` is not very intuitive in what it does.

Reply:
Thanks for pointing that out! I'll update it!